README.md

```diff
@@ -1 +1,111 @@
-# covid-tracker
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Covid Tracker
+
+## About
+
+This is a covid tracker app where you can find latest information about COVID in Brazil, total number of cases, today new confirmed cases and deaths and more, you can also search by state and check specific information about any state of Brazil.
+
+Our development flow included: performing git flow practices & using components with routers, life cycle methods,handling events and state management by Redux store,actions, reducers in [duck Patern](https://github.com/erikras/ducks-modular-redux) with unit and component tests.
+
+## Built With
+
+- Javascript
+- React & Redux
+- Redux Middleware (Thunk, Logger)
+- Bootstrap
+- Jest for testing
+
+## Live Demo
+
+[Live Demo Link](https://blissful-northcutt-e13aab.netlify.app/)
+
+## Presentation video
+
+[Watch video](https://www.loom.com/share/bff86c2832e348889c00439235d390d7)
+
+## Getting Started
+
+To get a local copy up and run the app; follow below steps.
+
+### Prerequisites
+
+If you are not familiar to use terminal, please check this [article](https://www.theodinproject.com/courses/web-development-101/lessons/command-line-basics-web-development-101)
+
+### Setup
+
+Open your terminal and run the following commands on your terminal.
+
+Make sure 'Javascript' is not blocked in your browser.
+
+### Install
+
+To get a local copy of the repository, run the following commands on your terminal.
+
+```
+cd <your_desktop>
+```
+
+```
+git clone git@github.com:yigitm/react-group-project.git
+```
+
+```
+cd react-group-project
+```
+
+### Usage
+
+To install the necessary Javascript packages to your project, run below commands.
+
+```
+npm install
+```
+
+To run the app, use below command.
+
+```
+npm start
+```
+
+It runs the app in the development mode and runs the app in [http://localhost:3000](http://localhost:3000).To view it open your default browser.
+
+### Run tests
+
+To run the app, use below command.
+
+```
+npm start
+```
+
+Launches the test runner in the interactive watch mode.\
+See the section about [running tests](https://facebook.github.io/create-react-app/docs/running-tests) for more information.
+
+### Deployment
+
+[Netlify](https://www.netlify.com/) is used for deployment.
+
+## Authors
+
+👤 **Jonathas Tavares**
+
+- GitHub: [jonathastavares](https://github.com/jonathastavares)
+- LinkedIn: [Jonathas Tavares](https://www.linkedin.com/in/jonathas-tavares)
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- Microverse
+
```

Comment:
- [ ] Your Acknowledgement section is missing appropriate credit to [Nelson Sakwa](https://www.behance.net/sakwadesignstudio) the [author of the original design](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)), as required by the [Creative Commons License](https://creativecommons.org/licenses/). Kindly add this.